src/Illuminate/Database/Connectors/MySqlConnector.php

```diff
@@ -16,6 +16,8 @@ public function connect(array $config)
     {
         $dsn = $this->getDsn($config);
 
+        $config = $this->formatSslOptions($config);
```

Comment:
This is a breaking change since you are changing the behaviour of the function

Reply:
This function just passes down the `sslkey`, `sslcert` and `sslrootcert` variable down to the options array in the right way. This doesn't break the current implementation.